packages/components/mention/src/mention.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div :class="ns.b()">
+    <el-input
+      v-bind="mergeProps(passInputProps, $attrs)"
+      ref="elInputRef"
+      :model-value="modelValue"
+      @input="handleInputChange"
+      @keydown="handleInputKeyDown"
+      @focus="handleInputFocus"
+      @blur="handleInputBlur"
+      @mousedown="handleInputMouseDown"
+    >
+      <template v-for="(_, name) in $slots" #[name]="slotProps">
+        <slot :name="name" v-bind="slotProps" />
+      </template>
+    </el-input>
+    <el-tooltip
+      ref="tooltipRef"
+      :visible="visible && (!!filteredOptions.length || loading)"
+      :popper-class="[ns.e('popper'), popperClass]"
+      :popper-options="popperOptions"
+      :placement="`${placement}-start`"
+      :fallback-placements="['bottom-start', 'top-start']"
+      effect="light"
+      pure
+      :offset="0"
+      :show-arrow="false"
```

Comment:
`effect`/`offset`..., it would be more convenient if these parameters were configurable to users?

Reply:
like the select component cascader component, effect does not support dark mode. other parameters have been modified